### ISSUE TITLE
Error when 'y' column has only zero values

### ISSUE THREAD

**Issue by Yiyio** [facebookincubator/prophet#211]
Hi,

I am trying to forecast (using R) sales of several thousand articles in a loop. When the loop goes through an article that has all zeroes in the 'y' column, it stops and throws the error:
```
 Error in FUN(X[[i]], ...) : Stan does not support NA (in y) in data
```

Inside each iteration of the loop I am doing this:
```
m <- prophet(data)

future <- make_future_dataframe(m, periods = 1, freq='w')

fcst <- predict(m, future)
```

I am also attaching a file with some sample data.

[data.xlsx](https://github.com/facebookincubator/prophet/files/1047937/data.xlsx)

**Comment by Ic3fr0g** [comment]
Prophet is built on Stan. As of now there is no way to handle missing data except throwing an error. From your end you can remove the articles from the loop if all their `y`s are 0's.

**Comment by bletham** [comment]
Thanks for attaching the data, that made this real easy. This is the same issue as #198. Having all 0's for y isn't an issue, and in fact having NAs in y isn't an issue either - Prophet filters them out. The issue is having rows with missing data in the ds column. The best thing would be to filter these rows out before passing the data into Prophet. In #198 we'll filter rows with missing ds.

**Comment by bletham** [comment]
This was fixed in #198.
